moved one step

richagi opened the pull request:
due to the redirect above the one step doesn't make sense in the configure section anymore

azurecla commented:
Hi __@enterprisebug__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@enterprisebug : Thanks for your contribution to the Azure documentation! The author, @steved0x, has been notified to review your proposed change.

steved0x commented:
@enterprisebug Thank you for your contribution and for catching this. 

@PRMerger these changes are good and are ready to be merged.

steved0x commented:
#sign-off

steved0x commented:
@ggailey777 these customer changes are good and are ready to be merged. Thank you.

richagi committed:
moved one step 

due to the redirect above the one step doesn't make sense in the configure section anymore
